command/catalog_command.go

```diff
@@ -0,0 +1,50 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = (*CatalogCommand)(nil)
+
+type CatalogCommand struct {
+	BaseCommand
+}
+
+func (c *CatalogCommand) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *CatalogCommand) Help() string {
+	helpText := `
+Usage: consul catalog <subcommand> [options] [args]
+
+  This command has subcommands for interacting with Consul's catalog. The
+  catalog should not be confused with the agent, although the APIs and
+  responses may be similar.
+
+  Here are some simple examples, and more detailed examples are available
+  in the subcommands or the documentation.
+
+  List all datacenters:
+
+      $ consul catalog datacenters
```

Comment:
I think just `datacenters` is fine since the only datacenter-related thing you can do is list them right now, if we added other actions later it could become `consul catalog datacenters list`.